doc/nrf/known_issues.rst

```diff
@@ -2739,6 +2739,15 @@ NCSDK-14015: Execution halts during boot
 NCSDK-13949: TF-M Secure Image copies FICR to RAM on nRF9160
   TF-M Secure Image copies the FICR to RAM address between ``0x2003E000`` and ``0x2003F000`` during boot on nRF9160.
 
+Samples
+*******
+
+.. rst-class:: v2-2-0
+
+  NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.
```

Comment:
```suggestion
NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.
  After adding the MSPL FEM Power splitter feature, the sample does not build with support for Skyworks FEM.
```
An issue always needs also the description after the summary.

Reply:
Ok, I completely changed description.